### ISSUE TITLE
how do i get to run the same task n times

### ISSUE THREAD

**Issue by aleksey-bykov** [Microsoft/vscode#30726]
in the latest version

![image](https://user-images.githubusercontent.com/937933/28225300-2ee097f8-68a0-11e7-9ff1-a9f3204039a5.png)

there is a number of tasks in the status bar that **implies there can be more than one task running at the same time**

![image](https://user-images.githubusercontent.com/937933/28225255-fef79d48-689f-11e7-8335-c6917d33bb9b.png)

yet when i try to run another task i get:

![image](https://user-images.githubusercontent.com/937933/28225269-12fc11de-68a0-11e7-9dce-4ee7c03fb1ac.png)

so how do i get to run more than one tasks if anything?

**Comment by chrmarti** [comment]
I think that refers to different task definitions. You can run the same task definition only once at a time. @dbaeumer can confirm.

**Comment by dbaeumer** [comment]
Yes, that is true. The same task can currently only be run once. Different tasks can be run in parallel. Keeping the request open to allow running the same task in parallel as well.

**Comment by dbaeumer** [comment]
Dups #32264 which already has a PR.
